bin/templates/cordova/lib/build.js

```diff
@@ -402,6 +440,16 @@ function parseOpts(options, resolvedTarget) {
                 case 'gradleArg':
                     ret.extraArgs.push(flagValue);
                     break;
+                case 'keystore':
+                case 'alias':
+                case 'keytorePassword':
+                case 'password':
+                case 'keystoreType':
+                    packageArgs[flagName] = flagValue;
+                    break;
+                case 'buildConfig':
+                    buildConfig = flagValue.replace(/"/g, '');
```

Comment:
Shouldn't need the `.replace()` here... Did you find it necessary for some reason?